src/openvic-simulation/map/MapInstance.hpp

```diff
@@ -44,6 +44,7 @@ namespace OpenVic {
 
 		bool setup(
 			BuildingTypeManager const& building_type_manager,
+			ModifierEffectCache const& modifier_effect_cache,
```

Comment:
`ModifierEffectCache` could also be forward declared here, assuming it currently comes only from `ProvinceInstance` and that gets replaced its own forward declaration